src/AutologoutManager.php

```diff
@@ -0,0 +1,164 @@
+<?php
+namespace Drupal\autologout;
+
+use Drupal\autologout\AutologoutHelperInterface;
+use Drupal\Core\Url;
+use Drupal\user\Entity\User;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Drupal\Core\Session\AnonymousUserSession;
+
+class AutologoutManager implements AutologoutManagerInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  function autologoutPreventJs() {
+    foreach (\Drupal::moduleHandler()->invokeAll('autologout_prevent') as $prevent) {
```

Comment:
Since this is a service, you don't need to call services directly with \Drupal::. Instead, you define them as depencencies at autologout.services.yml and then inject them in the __construct method of this class.